sharpy/utils/generator_interface.py

```diff
@@ -41,7 +38,7 @@ def generator_list_from_path(cwd):
     onlyfiles = [f for f in os.listdir(cwd) if os.path.isfile(os.path.join(cwd, f))]
 
     for i_file in range(len(onlyfiles)):
-        if onlyfiles[i_file].split('.')[-1] == 'py': # support autosaved files in the folder
```

Comment:
I remember there being a discussion of the issue of temporary files (created when editing the code) - should we leave it as is, unless there's a specific reason I don't know?

Reply:
Good spot! I just reversed this change